.github/pr-title-checker-config.json

```diff
@@ -0,0 +1,17 @@
+
+{
+    "LABEL": {
+      "name": "title needs adjustment",
+      "color": "EEEEEE"
+    },
+    "CHECKS": {
+      "prefixes": [
+        "chore: "      ],
```

Comment:
nit this file is formatted a bit strangely.

Reply:
removing that cleans it up a bit.